app/src/main/java/nerd/tuxmobil/fahrplan/congress/net/TLSSocketFactory.java

```diff
@@ -0,0 +1,69 @@
+package nerd.tuxmobil.fahrplan.congress.net;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+
+public class TLSSocketFactory extends SSLSocketFactory {
+
+    private SSLSocketFactory internalSSLSocketFactory;
+
+    TLSSocketFactory() throws KeyManagementException, NoSuchAlgorithmException {
+        SSLContext context = SSLContext.getInstance("TLS");
+        context.init(null, null, null);
```

Comment:
Shouldn't this be initialized with `sslContext.init(null, trustManagers, new SecureRandom());` such as in CustomHttpClient.java?